web3.js/module.d.ts

```diff
@@ -138,8 +137,28 @@ declare module '@solana/web3.js' {
     logs: Array<string> | null;
   };
 
+  export type PartiallyDecodedInnerInstruction = {
+    index: number;
+    instructions: PartiallyDecodedInstruction[];
+  };
```

Comment:
Oops, just noticed that this isn't quite right
```suggestion
  export type CompiledInnerInstruction = {
    index: number;
    instructions: CompiledInstruction[];
  };
```